src/context/AppContext.ts

```diff
@@ -2,12 +2,37 @@ import createUseContext from 'constate';
 
 import { preferredAxios, loggFeil, håndterApiRessurs } from './axios';
 import { AxiosRequestConfig, AxiosResponse, AxiosError } from 'axios';
-import { Ressurs, ApiRessurs } from '@navikt/familie-typer';
-
+import {
+    Ressurs,
+    ApiRessurs,
+    byggTomRessurs,
+    byggFeiletRessurs,
+    byggHenterRessurs,
+} from '@navikt/familie-typer';
 import { useState, useEffect } from 'react';
-import { autentiseringsInterceptor } from '../utils/autentisering';
+import { Person } from '../typer/person';
 
 const [AppProvider, useApp] = createUseContext(() => {
+    const [personData, settPersonData] = useState(byggTomRessurs<Person>());
```

Comment:
```suggestion
    const [sluttbruker, settSluttbruker] = useState(byggTomRessurs<Person>());
```

Usikker på navnet her, men trenger hvertfall ikke ta med Data. Det forstår man fra typen :)